Makefile

```diff
@@ -0,0 +1,130 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
```

Comment:
the image name need to be changed to the proper name?

Reply:
changed